core/driver/sources/src/main/scala/org/virtuslab/ideprobe/config/DriverConfig.scala

```diff
@@ -17,4 +18,9 @@ object DriverConfig {
       command: Seq[String] = Nil,
       timeout: FiniteDuration = 30.seconds
   )
+  case class ScreenConfig(
+      width: Int = 640,
```

Comment:
I'd say the defaults should be higher... low resolutions lead to weird errors like the ones I experience now with Xvfb screen being automatically opened at 640x480 (context menu out of screen etc.). I'd increase that values to sth well above 1000... Full HD maybe? I suppose that for Xvfb, the standard screen sizes (like Full HD) don't matter much anyway tho